Content.Server/Kitchen/EntitySystems/SharpSystem.cs

```diff
@@ -80,11 +80,12 @@ private void OnDoafterComplete(SharpButcherDoafterComplete ev)
         sharp.Butchering.Remove(ev.Entity);
 
         var spawnEntities = EntitySpawnCollection.GetSpawns(butcher.SpawnedEntities, _robustRandom);
-        var coords = Transform(ev.Entity).Coordinates;
+        var coords = Transform(ev.Entity).MapPosition;
         EntityUid popupEnt = default;
         foreach (var proto in spawnEntities)
         {
-            popupEnt = Spawn(proto, coords);
+            // distribute the spawned items randomly in a small radius around the origin
+            popupEnt = Spawn(proto, coords.Offset(_robustRandom.NextAngle().ToVec() * 0.25f));
```

Comment:
```suggestion
            popupEnt = Spawn(proto, coords.Offset(_robustRandom.NextVector2(0.25f));
```

Reply:
oh cool. This will also make it so they don't all spawn the same distance from the origin right? That's definitely in line with that I was thinking.